### ISSUE TITLE
How to build m66 branch?

### ISSUE THREAD

**Issue by Holo-Krzysztof** [ortclib/ortclib-sdk#16]
I've tried building the Robin/m66-msidl-new-mediasource branch because the current master is rather old and there's a lot of changes coming up. However, when trying to run the `prepare.bat` script it fails with the message `CRITICAL ERROR: COULD NOT CREATE SYMBOLIC LINK TO chromium\src\third_party\jsoncpp\source FROM ..\jsoncpp`.

Did the setup change from the steps described in the readme? The master branch builds without issues.

**Comment by Holo-Krzysztof** [comment]
I somehow got the prepare step to work by doing a deinit of all submodules, checking out the `releases/m66` branch, cleaning and initializing again (though I don't know why it worked because I'm pretty sure I've tried that before). The problem now is compilation itself, which fails because an import in a  build script can't be resolved.
Can anyone tell me what's causing this?

```
1>[92mWebrtc build is started. It will take couple of minutes.[0m
1>[92mWorking ...[0m
1>[92mx86[0m
1>[92mPlatform x86[0m
1>Visual Studio path is "C:\Program
1>Visual Studio 2017 Tools MSVC Version is 14.14.26428
1>Determining compiler options ...
1>CPU arhitecture is x64
1>Selected compiler option is amd64_x86
1>[93mBuilding libOrtc native libs[0m
1>[1/1900] STAMP obj/build/config/exe_and_shlib_deps.stamp
1>FAILED: obj/build/config/exe_and_shlib_deps.stamp
1>C:/Python27/python.exe ../../build/toolchain/win/tool_wrapper.py stamp obj/build/config/exe_and_shlib_deps.stamp
1>Traceback (most recent call last):
1>  File "../../build/toolchain/win/tool_wrapper.py", line 29, in <module>
1>    import win32file    # pylint: disable=import-error
1>ImportError: No module named win32file
1>[2/1900] STAMP obj/typedefs.stamp
1>FAILED: obj/typedefs.stamp
1>C:/Python27/python.exe ../../build/toolchain/win/tool_wrapper.py stamp obj/typedefs.stamp
1>Traceback (most recent call last):
1>  File "../../build/toolchain/win/tool_wrapper.py", line 29, in <module>
1>    import win32file    # pylint: disable=import-error
1>ImportError: No module named win32file
1>[3/1900] CXX obj/api/transport_api/transport.obj
1>[4/1900] CXX obj/logging/rtc_event_log_api/rtc_event_log.obj
1>[5/1900] CXX obj/api/video_frame_api/video_frame_buffer.obj
1>[6/1900] CXX obj/logging/ice_log/rtc_event_ice_candidate_pair.obj
1>[7/1900] CXX obj/api/video_frame_api/video_content_type.obj
1>[8/1900] CXX obj/api/video_frame_api/video_frame.obj
1>[9/1900] CXX obj/api/video_frame_api/video_timing.obj
1>[10/1900] CXX obj/logging/ice_log/icelogger.obj
1>ninja: build stopped: subcommand failed.
1>
1>EXEC : [91mCRITICAL error : Building libOrtc in C:\ortclib-sdk has failed"s[0m
1>
1>
1>[91mFAILURE: Building WebRtc library has failed[0m
1>
1>[93mTotal execution time: 00:00:02 (2s total)[0m
```

**Comment by Holo-Krzysztof** [comment]
Alright, I've finally got `Org.Ortc.Uwp` to build. I had to manually install pywin32 from [here](https://github.com/mhammond/pywin32/releases); not sure why the setup script doesn't do that when installing python (it'd be really nice if it did!).

One thing still doesn't work though, which is the PeerCC sample; there's a lot of errors in `ORTCStatsManager.cs`, probably due to API changes I guess.
